x-pack/plugins/security_solution/public/management/components/console/components/command_list.tsx

```diff
@@ -169,24 +214,35 @@ export const CommandList = memo<CommandListProps>(({ commands, display = 'defaul
         [commandsByGroup[0]?.helpGroupLabel ?? otherCommandsGroupLabel]: command,
       }));
     },
-    [otherCommandsGroupLabel]
+    []
   );
 
   const getTableColumns = useCallback(
     (commandsByGroup) => {
+      const groupLabel = commandsByGroup[0]?.helpGroupLabel ?? otherCommandsGroupLabel;
+      const groupTestIdSuffix = convertToTestId(groupLabel);
+
       return [
         {
-          field: commandsByGroup[0]?.helpGroupLabel ?? otherCommandsGroupLabel,
-          name: commandsByGroup[0]?.helpGroupLabel ?? otherCommandsGroupLabel,
+          field: groupLabel,
+          name: <div data-test-subj={getTestId('group')}>{groupLabel}</div>,
```

Comment:
I assume you wrap it as a div to pass data-test-subj, right? Wouldn't it be enough to pass data-test-subj as a separate property? Or if not, wrap groupLabel in React.Fragment?

Reply:
hmmm. `React.Fragment` accepting a test subj seems really strange since it does not actually output a DOM element.